tests/Stream.ts

```diff
@@ -0,0 +1,84 @@
+import '@relmify/jest-fp-ts';
+import { describe, jest, expect } from '@jest/globals';
+import { it } from '@fast-check/jest';
+
+import * as Stream from '../src/Stream';
+import * as fc from 'fast-check';
+import { pipe } from 'fp-ts/lib/function';
+
+import * as TE from 'fp-ts/lib/TaskEither';
+
+jest.setTimeout(5000);
+
+describe(Stream.map, () => {
+  it.prop([fc.array(fc.integer())])(
+    'can convert to an from an array',
```

Comment:
nit: can convert to a stream from an array`
```suggestion
    'can convert to and from an array',
```